test/e2e/utils/common.go

```diff
@@ -23,6 +24,16 @@ const (
 	truststoreSecretSuffix = "truststore"
 )
 
+func SkipForExposeType(test *testing.T, types ...ispnv1.ExposeType) {
+	exposeType := ispnv1.ExposeType(constants.GetEnvWithDefault("EXPOSE_SERVICE_TYPE", string(ispnv1.ExposeTypeNodePort)))
```

Comment:
We have this value in the constants already and could be polish a bit
`ExposeServiceType = constants.GetEnvWithDefault("EXPOSE_SERVICE_TYPE", "NodePort")`

Reply:
Good point! Updated.